src/lib.rs

```diff
@@ -66,15 +68,32 @@ use {
   prometheus_metrics::serve_prometheus,
 };
 
+#[derive(Parser, Debug)]
+struct CmdArgs {
+  #[arg(long, default_value_t = false)]
+  /// if you start multiple lemmy server instances set this to true on all but one of them
```

Comment:
```suggestion
  /// Disables running scheduled tasks.
  ///
  /// If you are running multiple Lemmy server processes,
  /// you probably want to disable scheduled tasks on all but one of the processes,
  /// to avoid running the tasks more often than intended.
```